Pipeline_example/scripts/exceptions.py

```diff
@@ -0,0 +1,10 @@
+"""
+Taken from Jenna's script from the Cellpainting repo
```

Comment:
Same recommendation from Mike above.

Reply:
Added a link in every doc. Thanks for the recommendation!